transformer/src/main/scala/uk/ac/wellcome/transformer/modules/StreamRecordProcessorFactoryModule.scala

```diff
@@ -40,21 +39,11 @@ class StreamsRecordProcessor(receiver: RecordReceiver)
   }
 }
 
-class StreamsRecordProcessorFactory(
+class StreamsRecordProcessorFactory @Inject()(
```

Comment:
> Generally, modules are only required for helping the injector instantiate classes that you don’t control. Otherwise, you would simply add the JSR-330 annotations (e.g., @Inject, @Singleton) directly to the class definition.